CMakeLists.txt

```diff
@@ -111,8 +111,17 @@ check_type_size(size_t SIZEOF_SIZE_T)
 include(TestBigEndian)  # Note: deprecated in CMake 3.20
 test_big_endian(WABT_BIG_ENDIAN)
 
+if(DEFINED ENV{WASM2C_CFLAGS})
+  set(WASM2C_CFLAGS "$ENV{WASM2C_CFLAGS}")
+endif ()
+  
 if (CMAKE_BUILD_TYPE STREQUAL "Debug")
   set(WABT_DEBUG 1)
+  set(WASM2C_CFLAGS "${WASM2C_CFLAGS} -g -O0")
+endif ()
+
+if (DEFINED CMAKE_OSX_SYSROOT)
+  set(WASM2C_CFLAGS "${WASM2C_CFLAGS} -isysroot ${CMAKE_OSX_SYSROOT}")
 endif ()
```

Comment:
WASM2C_CFLAGS is only used when running tests right?   Could we put all this stuff inside the `BUILD_TESTS` block to make this clear?

Reply:
Done (except for where we set the sanitizer flags -- I kept that one adjacent because that's where we know the full flags).